### DISCUSSION TITLE
Remnant: Keystone Research 7 needs a symbiont branch

### DISCUSSION THREAD

**Discussion by opusforlife2** [endless-sky/endless-sky#10638] [Lore]
If Dawn has already explained the symbionts to you in Remnant: Expanded Horizons Astral 1, then the symbiont talk here is redundant. So this mission text needs a "remnant symbionts info" branch too.

**Comment by TomGoodIdea** [comment]
For small changes, if you know exactly what should be changed, you can skip the issue/discussion stage and just open a PR.

**Reply by opusforlife2** [reply]
I can do text just fine. I'm not confident with conditionals or other game flow syntax.

**Comment by mOctave** [comment]
https://github.com/endless-sky/endless-sky/pull/10656
Do you happen to still have a save to test this with?

**Reply by opusforlife2** [reply]
Answered in PR.

**Comment by opusforlife2** [comment]
Fixed by #10656.
